src/pages/experiments/[experimentId]/plots-and-tables/heatmap/index.jsx

```diff
@@ -8,7 +8,7 @@ import { Vega } from 'react-vega';
 
 import DimensionsRangeEditor from '../components/DimensionsRangeEditor';
 import ColourbarDesign from '../components/ColourbarDesign';
-import LegendEditorSpecial from './components/LegendEditorSpecial';
```

Comment:
delete legendEditorspecial if it is no longer used

Reply:
Deleted.